lab_1/main.py

```diff
@@ -150,7 +186,11 @@ def write_to_file(path_to_file: str, content: list):
     """
     Writes the result in a file
     """
-    pass
+    if isinstance(path_to_file, str) and isinstance(content, list):
+
+        with open(path_to_file, 'w', encoding='utf-8') as file:
+            for concordance in content:
+                file.write(' '.join(concordance) + '\n')
```

Comment:
please, first put all concordances in one variable and then write it to file
it's not good to write to file at each iteration